### ISSUE TITLE
Enh: File browser

### ISSUE THREAD

**Issue by Thoulah** [yiisoft/yii2-bootstrap4#152]
We might also implement [File Browser](https://getbootstrap.com/docs/4.3/components/forms/#file-browser), but there are some caveats here.

- Label is not really used as label (or at least not how I would expect a label compared to other fields)
- The file input is the most gnarly of the bunch and requires additional JavaScript if you’d like to hook them up with functional <em>Choose file…</em> and selected file name text. They recommend [bs-custom-file-input](https://www.npmjs.com/package/bs-custom-file-input).

**Comment by samdark** [comment]
I'd wait with it till someone requests it.

**Comment by Thoulah** [comment]
Fair enough. I would like it, but not required. Just leaving following workaround here in case someone wants to play with it.

```
$fileLabel = Html::tag('label', 'Select a File', ['class' => 'custom-file-label']);

echo $form->field($model, 'fileField', [
    'inputTemplate' => Html::tag('div', '{input}' . $fileLabel, ['class' => 'custom-file']),
])->fileInput(['class' => 'custom-file-input']);
```

JS requirement not included.
